OrbitCaptureClient/CaptureEventProcessorTest.cpp

```diff
@@ -531,6 +545,382 @@ TEST(DISABLED_CaptureEventProcessor, CanHandleGpuJobs) {
   EXPECT_EQ(hw_excecution_timer.user_data_key(), actual_hw_execution_key);
 }
 
+GpuQueueSubmissionMetaInfo* CreateGpuQueueSubmissionMetaInfo(GpuQueueSubmission* submission,
+                                                             uint64_t pre_timestamp,
+                                                             uint64_t post_timestamp) {
+  GpuQueueSubmissionMetaInfo* meta_info = submission->mutable_meta_info();
+  meta_info->set_tid(2);
+  meta_info->set_pre_submission_cpu_timestamp(pre_timestamp);
+  meta_info->set_post_submission_cpu_timestamp(post_timestamp);
+  return meta_info;
+}
+
+void AddGpuCommandBuffer(GpuSubmitInfo* submit_info, uint64_t gpu_begin_timestamp,
+                         uint64_t gpu_end_timestamp) {
+  GpuCommandBuffer* command_buffer = submit_info->add_command_buffers();
+  command_buffer->set_begin_gpu_timestamp_ns(gpu_begin_timestamp);
+  command_buffer->set_end_gpu_timestamp_ns(gpu_end_timestamp);
+}
+
+void AddGpuDebugMarker(GpuQueueSubmission* submission, GpuQueueSubmissionMetaInfo* begin_meta_info,
+                       uint64_t marker_text_key, uint64_t begin_gpu_timestamp,
+                       uint64_t end_gpu_timestamp) {
+  GpuDebugMarker* debug_marker = submission->add_completed_markers();
+  Color* color = debug_marker->mutable_color();
+  color->set_alpha(1.f);
+  color->set_red(0.75f);
+  color->set_green(0.5f);
+  color->set_blue(0.25f);
+  debug_marker->set_depth(1);
```

Comment:
Shouldn't the color and the depth be parameters too?

Reply:
As above, I would at least add constants, especially since these values are reused in `ExpectDebugMarkerTimerEq`.

Reply:
Done.